org.fedoraproject.MediaWriter.json

```diff
@@ -0,0 +1,34 @@
+{
+    "id": "org.fedoraproject.MediaWriter",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "mediawriter",
+    "rename-appdata-file": "mediawriter.appdata.xml",
+    "rename-desktop-file": "mediawriter.desktop",
+    "finish-args": [
+        "--socket=x11",
+	"--device=dri",
+        "--filesystem=host",
+	"--share=network",
+	"--share=ipc",
+	"--system-talk-name=org.freedesktop.UDisks2"
+    ],
+    "modules": [
+        {
+            "name": "MediaWriter",
```

Comment:
flatpak-builder newer than 0.9.11 has a qmake buildssytem, you don't need to pass a configure script.